### DISCUSSION TITLE
"Some of these settings are managed by your organization." I got this message on some of the setting of my windows

### DISCUSSION THREAD

**Discussion by Lubra1234** [simeononsecurity/Windows-Optimize-Harden-Debloat#88] [General]
As the title says, some of the settings can be accessed due this "issue". So I'd like to know if it some natural due the Optimizations.
I am running Windows 11 Pro
![Screenshot 2024-03-02 085437](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/assets/146781713/804283c8-7bb4-4bad-8cc8-cab34f7cf8a9)

**Comment by simeononsecurity** [comment]
> As the title says, some of the settings can be accessed due this "issue". So I'd like to know if it some natural due the Optimizations. I am running Windows 11 Pro ![Screenshot 2024-03-02 085437](https://private-user-images.githubusercontent.com/146781713/309471246-804283c8-7bb4-4bad-8cc8-cab34f7cf8a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk0MDAzOTIsIm5iZiI6MTcwOTQwMDA5MiwicGF0aCI6Ii8xNDY3ODE3MTMvMzA5NDcxMjQ2LTgwNDI4M2M4LTdiYjQtNGJhZC04Y2M4LWNhYjM0ZjdjZjhhOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMwMlQxNzIxMzJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kNDU3YzMyMmEyYmUzNzE2YTlmMzM5NjVhYjg4MjAzN2IwMTY3Y2FhNGE1MzlkNzFkNmIwMGU3MTZiZTE1Zjg3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.w__SO2dJWk-Ad5_mQAWYKhfqp-f-jAB0s71NIYmvLuM)

Some of the settings are configured by GPO, this is an expected banner that will pop up.
Please see our [wiki](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/wiki)
If you run into anything that causes you issues.
